lms/static/scripts/frontend_apps/components/GroupConfigSelector.tsx

```diff
@@ -246,7 +246,7 @@ export default function GroupConfigSelector({
         onInput={(e: Event) =>
           onChangeGroupConfig({
             useGroupSet: (e.target as HTMLInputElement).checked,
-            groupSet: groupSet ?? null,
+            groupSet: (e.target as HTMLInputElement).checked ? groupSet : null,
```

Comment:
This shouldn't make a difference, since the value of `groupSet` is ignored here if `useGroupSet` is false:

https://github.com/hypothesis/lms/blob/5b027a40ce5d03217af7250214e3b38186c03703/lms/static/scripts/frontend_apps/components/FilePickerApp.tsx#L312

The `groupSet` prop that is passed to `FilePickerFormFields` is then mapped to either including a hidden form field with name `group_set`, or omitting that field:

https://github.com/hypothesis/lms/blob/5b027a40ce5d03217af7250214e3b38186c03703/lms/static/scripts/frontend_apps/components/FilePickerFormFields.tsx#L38

If we are going to require the group set field to be provided but with a null value to clear the group set, then I think we'd change the logic in `FilePickerFormFields` so that the `group_set` form field is always set, but with an empty value if no group set is chosen (ie. treating an empty string as `null`, since we can only pass string values here).